examples/servicebus-migration-test/step2/index.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+
+const exampleResourceGroup = new azure.core.ResourceGroup("example", {
+    location: "West Europe",
+    name: "my-servicebus",
+});
+const exampleNamespace = new azure.servicebus.Namespace("example", {
+    location: exampleResourceGroup.location,
+    name: "tfex_sevicebus_namespace",
+    resourceGroupName: exampleResourceGroup.name,
+    sku: "Standard",
+    tags: {
+        source: "example",
+    },
+});
+const serviceBusQueue = new azure.servicebus.Queue("example", {
```

Comment:
Should we convert all existing examples to the new namespaces? Might be a nice extra sanity check?

Reply:
yes, we should.  though that can come in another PR imo.  but it woudl just be good so anyone looking at it is pushed to the right path.